test/location.cc

```diff
@@ -13,173 +13,6 @@ using namespace valhalla::midgard;
 
 namespace {
 
-std::string make_json(float lat,
```

Comment:
this was the only use of the rapidjson parsing, no need to test something we dont use